### PR TITLE
NAS-109202 / 21.04 / Data Protection Dashboard

### PR DESCRIPTION
**Pull request by denysbutenko**

### CODE REVIEW

**Comment by dszidi** [comment]
I'm getting a traceback deleting a periodic snapshot task. It seems like the problem lies in how it tries to refresh the table after the task is deleted because once I refresh the page, the the task is gone.

image

**Comment by dszidi** [comment]
The navigation menu item and the page title are both Data Protection but the URL is still "tasks". We should make the URL match the title and navigation.

image

File: src/app/helptext/system/tunable.ts [hunk 1]
```diff
@@ -58,5 +58,10 @@ export const helptext_system_tunable = {
       "Enable this tunable. Unset to disable this tunable\
  without deleting it."
     )
+  },
+
+  tunable: {
+    add: T('Add Tunable'),
```
Comment (dszidi) [inline src/app/helptext/system/tunable.ts:64]: In Scale these are Sysctl not Tunable.

Comment (denysbutenko) [reply src/app/helptext/system/tunable.ts]: Fixed

**Review by dszidi — APPROVED** [review]
LGTM

**Pull request merged**